### ISSUE TITLE
💡RFC: Add bare node module support to Astro scripts

### ISSUE THREAD

**Issue by jasikpark** [snowpackjs/astro#997]
It'd be nice if you could do 
`<script>import "katex";</script>` as a script tag embedded in an Astro file-

it'd be pretty cool if bare modules were mapped to `/_astro/npm/modules/ms` or `/_astro/npm/modules/katex/v1/` in Astro, so that `import katex from "katex";` would be translated to `import katex from "/_astro/npm/modules/katex/v1/index.js";`

**Comment by drwpow** [comment]
This will work in the new compiler 🙂

**Comment by natemoo-re** [comment]
Confirming that this is fixed in the astro@next preview!

Will put this in our milestone to be closed when that release is stable.

**Comment by natemoo-re** [comment]
Following up on my previous comment, the team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
